src/libs/tools/filmstrip.c

```diff
@@ -1126,7 +1127,7 @@ static gboolean _lib_filmstrip_select_key_accel_callback(GtkAccelGroup *accel_gr
       dt_selection_select_filmroll(darktable.selection);
   }
 
-  gtk_widget_queue_draw(strip->filmstrip);
+  //gtk_widget_queue_draw(strip->filmstrip);
```

Comment:
why this? any bug with it?

Reply:
It is not needed, but I thought it would be better to use control signals e.g. DT_SIGNAL_COLLECTION_CHANGED instead of  calling it by myself. So perhaps the number of redraws are reduced and I know whether all signals are raised properly.